### ISSUE TITLE
One test fails

### ISSUE THREAD

**Issue by yurivict** [BlueBrain/HighFive#634]
```
  8/268 Test   #8: Test group properties .........................................Subprocess aborted***Exception:   0.01 sec
Assertion failed: (mesg->raw_size < 65536), function H5O_msg_flush, file /wrkdirs/usr/ports/science/hdf5/work/CMake-hdf5-1.12.2/hdf5-1.12.2/src/H5Omessage.c, line 2054.
Filters: Test group properties

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
tests_high_five_base is a Catch v2.13.9 host application.
Run with -? for options

-------------------------------------------------------------------------------
Test group properties
-------------------------------------------------------------------------------
/disk-samsung/freebsd-ports/science/highfive/work/HighFive-2.5.0/tests/unit/tests_high_five_base.cpp:218
...............................................................................

/disk-samsung/freebsd-ports/science/highfive/work/HighFive-2.5.0/tests/unit/tests_high_five_base.cpp:218: FAILED:
  {Unknown expression after the reported line}
due to a fatal error condition:
  SIGABRT - Abort (abnormal termination) signal

===============================================================================
test cases: 1 | 1 failed
assertions: 3 | 2 passed | 1 failed

```

Version: 2.5.0
clang-14
FreeBSD 13.1 STABLE

**Comment by matz-e** [comment]
Duplicate of #550. I'm sorry, we're still waiting on upstream to provide a fix in HDF5.
